crates/fluvio-cluster/src/cli/shutdown.rs

```diff
@@ -94,6 +94,7 @@ impl ShutdownOpt {
                     "Removed spu monitoring socket: {SPU_MONITORING_UNIX_SOCKET}"
                 ));
             }
+            Err(io_err) if io_err.kind() == std::io::ErrorKind::NotFound => {}
```

Comment:
is this supposed to result in a silent error, no log?

Reply:
I think so, it was printing err that `not find the socket` when there is no fluvio running.

Reply:
added a debug! log, what do you think?